changelog/unreleased/kong/resty-simdjson.yml

```diff
@@ -0,0 +1,5 @@
+message: |
+  Introduced a new yieldable JSON library lua-resty-simdjson,
+  which will improve the latency dramatically.
```

Comment:
```suggestion
  Introduced a yieldable JSON library `lua-resty-simdjson` to replace the `cjson` library,
  which would improve the latency significantly.
```

Reply:
Partly accepted, but in this PR we will not replace cjson completely.